pkg/k8s/service.go

```diff
@@ -123,10 +149,11 @@ func ParseService(svc *slim_corev1.Service, nodeAddressing datapath.NodeAddressi
 		lbSrcRanges = append(lbSrcRanges, cidrStringTrimmed)
 	}
 
-	svcInfo := NewService(clusterIP, svc.Spec.ExternalIPs, loadBalancerIPs,
+	svcInfo := NewService(clusterIPs, svc.Spec.ExternalIPs, loadBalancerIPs,
```

Comment:
My suggestion would be to order the `clusterIPs` here to avoid ordering them every time `String()` is executed.